src/urbit/app/chess.hoon

```diff
@@ -597,23 +605,33 @@
             challenges-sent  (~(del by challenges-sent) src.bowl)
           ==
       ==
+    ::
+    ::  handle actions from opponent player
+    ::  TODO: what is the weird subscription?
```

Comment:
I'm not sure. The context makes me think that it should mean "unsubscribe instead of doing nothing", but Ray knew that to-do tasks should start with `XX` and used that notation in other comments, so it seems like he meant "leave it alone". Let's leave it alone for now, as maybe like the `@dau` we'll discover the reasoning for it while poking around.